### PR TITLE
attempted to give better descriptions for the insights documents. Fee…

### PR DESCRIPTION
**Pull request by ChristineWuerth**
…l free to edit.

### CODE REVIEW

File: docs/frontend-templates/homepage.md [hunk 1]
```diff
@@ -246,7 +269,16 @@ insights:
...
       # You MUST wrap this in single quotation marks (ie. ' '), since markdown can be used to enter this information. To create a heading, use three hashes (ie. ###).
-      'Information regarding the design update, taken place in November 2023'
+      'Information regarding the design update, taken place in November 2023. 
+      
+      Changes included:
+
+      - a visual refresh (larger font sizes, more spacing between sections, making the design clearer on mobile devices)
+      
+      - changes to the page structure based on what we know about what users do and what they expect (removing unused sections, 
+      - showing list of topics as a list vs grid, adding bullets to the list of popular links)
```
Comment (nnagewad) [inline docs/frontend-templates/homepage.md:279]: Is this supposed to be another list item? Currently not rendered as that.

**Review by nnagewad — CHANGES_REQUESTED** [review]
If it's a separate item you'll need to update spacing

File: docs/frontend-templates/homepage.md (hunk 1 above)
Comment (ChristineWuerth) [reply docs/frontend-templates/homepage.md]: thanks for spotting. That should not be there. I tried to weasle my way through github to amend this but got confused. Not sure if the amend came through.

Comment (nnagewad) [reply docs/frontend-templates/homepage.md]: Right on lets work on this together

**Review by nnagewad — APPROVED** [review]

**Pull request merged**